compiler/rustc_session/src/config/cfg.rs

```diff
@@ -142,6 +142,10 @@ pub(crate) fn disallow_cfgs(sess: &Session, user_cfgs: &Cfg) {
             | (sym::target_has_atomic, Some(_))
             | (sym::target_has_atomic_equal_alignment, Some(_))
             | (sym::target_has_atomic_load_store, Some(_))
+            | (sym::target_has_reliable_f16, Some(_))
+            | (sym::target_has_reliable_f16_math, Some(_))
+            | (sym::target_has_reliable_f128, Some(_))
+            | (sym::target_has_reliable_f128_math, Some(_))
```

Comment:
We have a test for those disallowed cfgs, would be great if you could add those in it and keep the list up-to-date.
https://github.com/rust-lang/rust/blob/496145b9cc023aef4bb1f16c0964a53d0da36c88/tests/ui/cfg/disallowed-cli-cfgs.rs#L1-L10

Reply:
I've updated the test